packages/block-library/src/embed/core-embeds.js

```diff
@@ -338,4 +339,25 @@ export const others = [
 		},
 		patterns: [ /^https?:\/\/wordpress\.tv\/.+/i ],
 	},
+	{
+		name: 'core-embed/amazon-kindle',
+		settings: {
+			title: 'Amazon Kindle',
+			icon: embedAmazonIcon,
+			keywords: [ __( 'ebook' ) ],
+			description: __( 'Embed Amazon Kindle content.' ),
```

Comment:
we should add `responsive: false,` here too, as Amazon delivers the content in an iframe with fixed dimensions, but it doesn't conform to the usual video aspect ratios that the responsive styles use, so we get a load of extra padding that we don't want with responsive styles enabled.

Reply:
Thanks for your feedback, I added it.